pages/api/v4/[network]/metadata/token.js

```diff
@@ -38,6 +39,18 @@ const api = async (req, res) => {
         break;
     }
 
+    if (req.method === "POST") {
+      const body = JSON.parse(JSON.stringify(req.body));
+      let metadata = parse(body);
```

Comment:
maybe add a check that the method is opensea (similar to how you did with the slug) as you only support opensea parsing